WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsDetails.swift

```diff
@@ -90,13 +99,75 @@ private struct CustomFieldRow: View {
             Spacer()
         }
         .padding([.top, .bottom], Constants.hStackPadding)
+        .padding(.horizontal, Constants.hStackPadding)
```

Comment:
Nit: this can be combined into `.padding(Constants.hStackPadding)` to set the same padding for all edges.

Reply:
Updated as part of https://github.com/woocommerce/woocommerce-ios/pull/13639/commits/1706a3fa8a3a6a4e9abe47892321f5971b050da2